### PR TITLE
CAT-1307 added new resources

### PR DESCRIPTION
**Pull request by thmq**
linked to #1179 

added new string-resources.

### CODE REVIEW

**Comment by avihric** [comment]
ok to test

File: catroid/res/values-da/strings.xml [hunk 1]
```diff
@@ -0,0 +1,548 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<!--Generated by crowdin.com-->
+<!--
+  ~ Catroid: An on-device visual programming system for Android devices
+  ~ Copyright (C) 2010-2014 The Catrobat Team
```
Comment (avihric) [inline catroid/res/values-da/strings.xml]: Update to 2015, as #1194 is merged

File: catroid/res/values-hi/strings.xml [hunk 2]
```diff
@@ -0,0 +1,559 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<!--Generated by crowdin.com-->
+<!--
+  ~ Catroid: An on-device visual programming system for Android devices
+  ~ Copyright (C) 2010-2014 The Catrobat Team
```
Comment (avihric) [inline catroid/res/values-hi/strings.xml]: same here, update to 2015

File: catroid/res/values-no/strings.xml [hunk 3]
```diff
@@ -0,0 +1,555 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<!--Generated by crowdin.com-->
+<!--
+  ~ Catroid: An on-device visual programming system for Android devices
+  ~ Copyright (C) 2010-2014 The Catrobat Team
```
Comment (avihric) [inline catroid/res/values-no/strings.xml]: again, update to 2015

File: catroid/res/values-th/strings.xml [hunk 4]
```diff
@@ -0,0 +1,547 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<!--Generated by crowdin.com-->
+<!--
+  ~ Catroid: An on-device visual programming system for Android devices
+  ~ Copyright (C) 2010-2014 The Catrobat Team
```
Comment (avihric) [inline catroid/res/values-th/strings.xml]: here to

File: catroid/res/values-ur/strings.xml [hunk 5]
```diff
@@ -0,0 +1,555 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<!--Generated by crowdin.com-->
+<!--
+  ~ Catroid: An on-device visual programming system for Android devices
+  ~ Copyright (C) 2010-2014 The Catrobat Team
```
Comment (avihric) [inline catroid/res/values-ur/strings.xml]: another one

**Comment by avihric** [comment]
You will have to pull new strings from Crowdin, as some new strings are introduced with #1183

**Comment by thmq** [comment]
ok to test

**Comment by avihric** [comment]
Please review again when #1184 is merged.

**Comment by LauraBebek** [comment]
Hallo Ajdin,
Ich schaffs heut leider nit zum meeting...

Liebe Grüsse,
Laura

**Comment by obusher** [comment]
:O

**Comment by LauraBebek** [comment]
Hi Ajdin, 
Ich schaffs heut nicht zum meeting...

Liebe Grüsse,
Lauta

**Comment by avihric** [comment]
@LauraBebek :consfused: why aren't you writing me an email?

**Pull request closed without merging**